app-api/src/main/java/org/openmrs/eip/app/management/entity/SenderSyncMessage.java

```diff
@@ -55,6 +55,10 @@ public enum SenderSyncMessageStatus {
 	@Access(AccessType.FIELD)
 	private Date dateSent;
 	
+	@Column(name = "event_date")
+	@Access(AccessType.FIELD)
```

Comment:
It's pointless to set access to field, yet the property has a getter/setter, please remove @Access